api/controllers/UserController.js

```diff
@@ -141,8 +141,13 @@ module.exports = {
                       if (req.user && user.name === req.user.name) {
                         user.isMod = req.user.isMod;
                       }
-
-                      user.references = references;
+                      var publicReferences=references;
+                      publicReferences.forEach(function(entry) {
+                        //If the current user is not the user that submitted the trade, remove the private notes before sending the trade info.
+                        if (!req.user||req.user.id!==entry.user)
```

Comment:
I'm going to set up a code style to import, but can we get spaces here please, I can't read things like that properly.

No need for spaces after (, but before and after || and !== would be good.